openstack/cce/v3/addons/delete.go

```diff
@@ -0,0 +1,19 @@
+package addons
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+// Delete will permanently delete a particular addon based on its unique ID.
+func Delete(client *golangsdk.ServiceClient, id, clusterId string) (err error) {
+	_, err = client.Delete(fmt.Sprintf("https://%s.%s", clusterId, client.ResourceBaseURL()[8:])+
+		strings.Join([]string{"addons", id + "?cluster_id=" + clusterId}, "/"), &golangsdk.RequestOpts{
```

Comment:
Nope, joining is an overcomplication here. Either move to a function, as it was before or use `fmt.Sprintf`